sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpClient.cs

```diff
@@ -246,7 +245,7 @@ internal class AmqpClient : TransportClient
 
                         if (!ManagementLink.TryGetOpenedObject(out link))
                         {
-                            link = await ManagementLink.GetOrCreateAsync(UseMinimum(ConnectionScope.SessionTimeout, tryTimeout.CalculateRemaining(stopWatch.GetElapsedTime())), cancellationToken).ConfigureAwait(false);
+                            link = await ManagementLink.GetOrCreateAsync(tryTimeout, cancellationToken).ConfigureAwait(false);
```

Comment:
The goal is for the `timeout` to cover the end-to-end interval.   In some cases, we are creating a stopwatch and then decrementing the timeout after not doing anything.   The first operation in a series or when we have a single operation should just use the timeout.

The two in here are mistakes - I overlooked that call to `AcquireAccessTokenAsync` - which is potentially a service call.   I need to the watch back here so that we properly deduct the time that it takes to perform auth.

Reply:
Ok I understand. That makes sense! Thank you!